Modify dev based on comments from nf-core reviewers (#471)
nf-core/airrflow: docs/usage/genotyping_tutorial.md

bash airrflow_genotyping_codespace.sh
```

> The estimated running time is 15 minutes. We have also provided the [results on Zenodo](https://zenodo.org/records/20450474) in case you do not have enough time to run the pipeline yourself.

Reviewer: ```suggestion
The estimated running time is 15 minutes. We also uploaded the tutorial [results on Zenodo](https://zenodo.org/records/20450474).
```
